packages/stems/src/hooks/useGlobal.ts

```diff
@@ -23,6 +21,10 @@ export const useGlobal = <T>(
   name: string,
   initialValue: T
 ): [() => T, (mutator: (cur: T) => void) => void] => {
+  if (window.AudiusStems === undefined) {
```

Comment:
seems like we can just drop useGlobal?

Reply:
Yes! I converted the one place we used it to use a singleton variable instead bc useGlobal relies on window